DEVELOPERS.md

```diff
@@ -0,0 +1,31 @@
+# Developers Notes
+
+## Setup
+
+- Please use the `make git_hooks` to setup the pre-commit hooks
+
+## Linting
+
+At the moment we are using pyright pegged at version 1.1.294
+
+## VS Code
+
+Probably fair to say most of use VS Code so its worth writing something here
+about our shared setup
+
+### Linting
+
+We use pyright for linting (see .pre-commit-config.yml) but [pyright recommend
+that you use Pylance when using VS Code](https://microsoft.github.io/pyright/#/installation?id=vs-code).
+
+Here is a snippet of the recommended:
```

Comment:
Again, might be good to specify which settings to add this to in VS code e.g., Ctrl + Shift + P -> Preferences: Open User Settings (JSON).

Reply:
I'm loathed to put in instructions on using VSCode as this can change - bit of a tight couple - but happy to be convinced otherwise